src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -41,8 +41,13 @@
     var existsOptionInSelectControl = function (selectControl, value) {
         var options = selectControl.get(selectControl.length - 1).options;
         for (var i = 0; i < options.length; i++) {
-            if (options[i].value === value.toString()) {
```

Comment:
Why don't you use:
```
if (!value || options[i].value === value.toString()) {
    return false;
}
```
instead of:
```
if(value){
    if (options[i].value === value.toString()) {
        //The value is nor valid to add
        return false;
    }
}
else{
    return false;
}
```